sktalk/corpus/parsing/cha.py

```diff
@@ -0,0 +1,69 @@
+import re
+import pylangacq
+from ..conversation import Conversation
+from ..utterance import Utterance
+from .parser import Parser
+
+
+class ChaParser(Parser):
+    def parse(self, file):
+        """Parse conversation file in Chat format
+
+        Args:
+            file (str): path to file in .cha format
+
+        Returns:
+            Conversation: A Conversation object representing the conversation in the file.
+        """
+        chatfile = pylangacq.read_chat(file)
+        chat_utterances = chatfile.utterances(by_files=False)
+
+        metadata = ChaParser._get_metadata(chatfile)
+
+        utterances = [ChaParser._to_utterance(
+            row, file) for row in chat_utterances]
+
+        return Conversation(utterances, metadata)
+
+    @staticmethod
+    def _to_utterance(row, file):
+        utterance = Utterance(
+            participant=row.participant,
+            time=row.time_marks,
+            utterance=str(row.tiers),
+            source=file,
+        )
+        utterance.begin, utterance.end = ChaParser._split_time(utterance.time)
+        utterance.utterance = ChaParser._clean_utterance(utterance.utterance)
+        return utterance
+
+    @staticmethod
+    def _get_metadata(chatfile):
```

Comment:
Some design concerns here (to be discussed):
1. I believe this should not be a static method, but a class method which would allow potential sub-classes in the future to override it.
2. By convention, method names using `get` imply a primitive operation returning a property of the object. Therefore, I would prefer a name like `_read_metadata(chatfile)`.

Also, I think the `chatfile` argument should have a type hint because I think it is not obvious whether it expects a file name or perhaps `Path` object or a file handler.

```suggestion
    def _extract_metadata(self, chatfile: str):
```